src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,32 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+
+import { Button } from './Button';
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
Let's turn this off for stories in the ESLint config `overrides`.

Reply:
yessir